Assets/Scripts/Tile.cs

```diff
@@ -45,4 +44,41 @@ public void OnPointerClick(PointerEventData eventData)
         BuildManager.instance.tileToBuildOn = this;
         BuildPanel.instance.Show(this);
     }
+
+    void Start()
```

Comment:
Start(), Update(), Awake(), LateUpdate(), FixedUpdate() z reguły umieszczamy przed pozostałymi metodami. Są to metody wszystkich obiektów, które dziedziczą po MonoBehaviour.